Makefile

```diff
@@ -420,27 +420,27 @@ pkg/version/_buildinfo.json: bin/buildhelper
 
 update-buildinfo: pkg/version/_buildinfo.json
 
-bin/buildhelper:
+bin/buildhelper: tools/buildhelper/buildhelper.go
```

Comment:
ok, this is more correct, but let's repeat ourselves just once then:
```
bin/buildhelper: tools/buildhelper/buildhelper.go
	@go build -o $@ $<
```
https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html

Reply:
Nice. TIL